### PR TITLE
Added support for Century

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
For issues #725 

This PR : century added to library

### CODE REVIEW

File: dateparser/data/date_translation_data/en.py [hunk 1]
```diff
@@ -189,6 +189,13 @@
         "in 1 decade": [
             "next decade"
         ]
+          "1 century ago": [
```
Comment (noviluni) [inline dateparser/data/date_translation_data/en.py:192]: It should be a comma here.

**Review by noviluni — CHANGES_REQUESTED** [review]
The pipelines are failing because the `en.py` file is bad formatted (missing comma).

File: dateparser/data/date_translation_data/en.py [hunk 2]
```diff
@@ -188,7 +188,14 @@
         ],
         "in 1 decade": [
             "next decade"
-        ]
+        ],
+          "1 century ago": [
```
Comment (noviluni) [inline dateparser/data/date_translation_data/en.py:192]: the alignment for "1 century ago" is not correct, it should be moved to the left

File: dateparser/data/date_translation_data/en.py [hunk 3]
```diff
@@ -188,7 +188,14 @@
...
+            "last century",
+            "this century"
+        ],
+        "in 1 century": [
+            "next century"
+        ]	        ]
```
Comment (noviluni) [inline dateparser/data/date_translation_data/en.py:198]: There are two `]` instead of one. This has to be fixed in the line 282 too.

**Review by noviluni — CHANGES_REQUESTED** [review]
Hi @NEERAJAP2001, could you check this again?

**Comment by NEERAJAP2001** [comment]
@noviluni sir, I have fixed errors in ```en.py``` Why is it still failing?

**Comment by noviluni** [comment]
Hi @NEERAJAP2001 , there are a lot of different things failing

1. 
One thing that it's failing is the pep8 checks:

```
dateparser/dateparser/freshness_date_parser.py:152:1: W293 blank line contains whitespace
dateparser/dateparser/data/date_translation_data/en.py:198:10: W291 trailing whitespace
dateparser/dateparser/data/date_translation_data/en.py:282:10: W291 trailing whitespace
dateparser/data/date_translation_data/en.py:788:6: E131 continuation line unaligned for hanging indent
dateparser/dateparser/languages/dictionary.py:13:66: E231 missing whitespace after ','
dateparser/tests/test_data.py:21:49: E231 missing whitespace after ','
```

2. 
Another test failing is this: `test_dateparser_data_integrity`, and this is failing because you wrote the file `en.py` manually instead of generating it with `python dateparser_scripts/write_complete_data.py`.

3. 
In the `test_relative_future_dates` parameters you wrote `ago`instead of `in_future`.

4. 
In 

```
        if 'century' in kwargs:
            kwargs['years'] = 100 * kwargs['century'] + kwargs.get('years', 0)
```

I think that the keyword instead of `century` it should be `centurys`, and you forgot the `del kwargs['centurys']` (like in `decade`).

----------
This won't fix all the issues, but I prefer you to fix these things first before checking the other things.

On the other side, you should be able to run the tests locally to see if it's working. To do it you can:

`pip install tox`

and then:

`tox`

to run all the tests, or:

`tox -e py3 -- tests/test_freshness_date_parser.py` (for example)

to run some specific tests

Let me know if you have any doubt.

**Comment by NEERAJAP2001** [comment]
@noviluni okay, sir, I'll come back with a new PR !!

**Pull request closed without merging**